packages/email-templates/src/index.ts

```diff
@@ -30,7 +30,22 @@ export const EmailTemplates: Record<Partial<Template>, EmailTemplateProps> = {
   debug,
   transferCode,
   keyAirdropped,
-  ...LockTemplates,
 }
 
-export const getEmailTemplate = (template: Template) => EmailTemplates[template]
+const templates = {}
+Object.keys(LockTemplates).forEach((template: any) => {
+  // @ts-ignore
+  templates[template.toLowerCase()] = LockTemplates[template]
+})
+
+Object.keys(EmailTemplates).forEach((template: string) => {
+  // @ts-ignore
+  templates[template.toLowerCase()] = EmailTemplates[template]
+})
+
+const getEmailTemplate = (template: Template) => EmailTemplates[template]
+// @ts-ignore
+const getLockTemplate = (template: any) => LockTemplates[template]
```

Comment:
Why are we introducing a new method here? I thin this will be error prone. Let's just have `getEmailTemplate` return *all* templates (generic AND lock specific).